### PR TITLE
Update multi-language checkout.md

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-2338](https://jira.bigcommerce.com/browse/DEVDOCS-2338)

## What changed?
removed div tags in order to fix the text wrapping issue

### CODE REVIEW

File: docs/stencil-docs/localization/multi-language-checkout.md [hunk 1]
```diff
@@ -58,6 +47,7 @@ lineNumbers: true
                 {{/if}}
             </a>
         </h2>
+      </h1>
```
Comment (aglensmith) [inline docs/stencil-docs/localization/multi-language-checkout.md:50]: This isn't quiet right. 

Should be like this: 

```html
<h1 class="is-srOnly">{{lang 'checkout.title'}}</h1>
```

**Review by aglensmith — CHANGES_REQUESTED** [review]

File: docs/stencil-docs/localization/multi-language-checkout.md [hunk 2]
```diff
@@ -134,13 +123,6 @@ BigCommerce's Optimized One-Page Checkout will currently translate displayed **C
 
 ## Limits on translation
 
-* The translation of your theme's content consists of the language JSON files in your `lang` subdirectory and the key-value pairs for the parameters (beyond **Country/State**) that you choose to translate.
-* Stencil's multi-language capabilities are limited to the strings that you specify within the theme. The Stencil framework does not currently translate content rendered from a store's database; for example, products' names.
```
Comment (uwikaiddi) [inline docs/stencil-docs/localization/multi-language-checkout.md:138]: Are we deleting the two bullets below?

Comment (uwikaiddi) [reply docs/stencil-docs/localization/multi-language-checkout.md:138]: ^ @traci707

**Review by uwikaiddi — CHANGES_REQUESTED** [review]

File: docs/stencil-docs/localization/multi-language-checkout.md (hunk 2 above)
Comment (traci707) [reply docs/stencil-docs/localization/multi-language-checkout.md:138]: I am fixing the duplication. Limits on translation is listed twice.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**